diff -uNr openmediavault/usr/share/php/openmediavault/system.inc openmediavault.patched/usr/share/php/openmediavault/system.inc
--- openmediavault/usr/share/php/openmediavault/system.inc	2014-11-01 01:48:49.711205000 +0100
+++ openmediavault.patched/usr/share/php/openmediavault/system.inc	2014-11-04 19:51:23.398917028 +0100
@@ -4265,7 +4265,7 @@
 	 * @return An array of system user names, otherwise FALSE.
 	 */
 	public static function enumerate() {
-		$cmd = "export LANG=C; getent passwd";
+		$cmd = "export LANG=C; " . $GLOBALS['OMV_GETENT_PASSWD'] ;
 		@OMVUtil::exec($cmd, $output, $result);
 		if ($result !== 0)
 			return FALSE;
